Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -510,6 +529,26 @@ Vector<WeakPtr<RenderedDocumentMarker>> DocumentMarkerController::markersFor(Nod
     return result;
 }
 
+// Finds any document markers of the specified types that contain a collapsed range matching `range`,
+// and applies `function` to them.
+void DocumentMarkerController::applyToCollapsedRangeMarker(const SimpleRange& range, OptionSet<DocumentMarkerType> types, Function<bool(Node&, RenderedDocumentMarker&)>&& function)
```

Comment:
Should this early return if `types` doesn't contain `DocumentMarkerType::TransparentContent`?

Reply:
I don't think it's needed, since you can't insert a collapsed DocumentMarker that isn't of that type in the first place, so this function will never do anything for other types